RadarSDK/RadarAPIHelper.m

```diff
@@ -129,9 +134,17 @@ - (void)requestWithMethod:(NSString *)method
 
                     res = (NSDictionary *)resObj;
 
-                    [[RadarLogger sharedInstance]
-                        logWithLevel:RadarLogLevelDebug
-                             message:[NSString stringWithFormat:@"📍 Radar API response | method = %@; url = %@; statusCode = %ld; res = %@", method, url, (long)statusCode, res]];
+                    if (params && [params objectForKey:@"replays"]) {
+                        NSArray *replays = [params objectForKey:@"replays"];
+                        [[RadarLogger sharedInstance]
+                            logWithLevel:RadarLogLevelDebug
+                                 message:[NSString stringWithFormat:@"📍 Radar API response | method = %@; url = %@; statusCode = %ld; res = %@; latency = %f; replays = %lu", method, url, (long)statusCode, res, latency, (unsigned long)replays.count]];
```

Comment:
add units for `latency`?

Reply:
Hmmm, the other values here don't have units, and it sort of breaks the convention going on here. I noted in the comment you suggested that the value is in seconds. Changing this string to something like `latency (s)=` seems non-ideal, but open to switching if you feel strongly.